sdk/keyvault/Azure.Security.KeyVault.Secrets/aot/Azure.Security.KeyVault.Secrets.Aot.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net7.0</TargetFramework>
+    <PublishAot>true</PublishAot>
+    <TrimmerSingleWarn>false</TrimmerSingleWarn>
+    <IsClientLibrary>false</IsClientLibrary>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <TrimmerRootAssembly Include="Azure.Security.KeyVault.Secrets" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="../../../core/Azure.Core/src/Azure.Core.csproj" />
+    <ProjectReference Include="../src/Azure.Security.KeyVault.Secrets.csproj" />
+  </ItemGroup>
```

Comment:
```suggestion
  </ItemGroup>
  
  <ItemGroup>
      <!-- Update this dependency to its latest, which has all the annotations -->
      <PackageReference Include="Microsoft.Extensions.Logging.Configuration" Version="8.0.0-preview.7.23375.6" />
  </ItemGroup>
```

This isn't 100% necessary but helps get the most up-to-date set of warnings.

Reply:
We should probably add that to `eng/Packages.Data.props` so we can manage it centrally. Rather, `Update` vs. `Include`.